### PR TITLE
Add some commenting platforms

### PR DESCRIPTION
**Pull request by opxop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1027,6 +1027,8 @@ Table of Contents
   * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience. 
   * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
   * [Disqus](https://disqus.com/) - Disqus is a networked community platform used by hundreds of thousands of sites all over the web.
+  * [Gitalk](https://gitalk.github.io/) - Gitalk is a modern comment component based on Github Issue and Preact.
```
Comment (ripienaar) [inline README.md:1030]: I wouldnt say this is a *aaS, this is a library that uses github as a backend, it's nice but its not a *aaS.

File: README.md [hunk 2]
```diff
@@ -1027,6 +1027,8 @@ Table of Contents
   * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience. 
   * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
   * [Disqus](https://disqus.com/) - Disqus is a networked community platform used by hundreds of thousands of sites all over the web.
+  * [Gitalk](https://gitalk.github.io/) - Gitalk is a modern comment component based on Github Issue and Preact. 
+  * [Valine](https://valine.js.org/) - Valine is a fast, simple & powerful comment system.
```
Comment (ripienaar) [inline README.md:1031]: Think same comment here really

**Comment by opxop** [comment]
I admit, I'm not really familiar with the real *aaS technology, thanks for correcting me

**Comment by ripienaar** [comment]
No worries, *aaS tends to be a service offered by some company who runs infrastructure behind and you pay them to run it.  Now your github backed comment system is a thin line between - but I think its more just a library (ab)using something else. What *aaS isn't is something you host yourself.

Appreciate the contribution though!

**Pull request closed without merging**